library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -165,6 +165,9 @@ $skin-with-icons: true !default;
 $icon-size: 20px !default;
 $tool-menu-icon-on-left: true !default;
 
+/* Icon Overrides */
```

Comment:
I understood that `_defaults.scss` was where we setup the defaults, but we encouraged people to create a customisation file that overrides the defaults rather than actually changing the `_defaults.scss` file directly. Should the icon overrides file go into a sample customisation file instead?